xknx/telegram/apci.py

```diff
@@ -730,7 +755,14 @@ def from_knx(self, raw: bytes) -> None:
 
     def to_knx(self) -> bytes:
         """Serialize to KNX/IP raw data."""
-        return encode_cmd_and_payload(self.CODE)
+        additional_flags = None
+        if self.sequqence_number:
+            additional_flags = APCIAdditionalFlags.NUMBERED_DATA_PACKET
```

Comment:
is this supposed to be `None` if sequence_number is 0? 
you could define `self.additional_flags = APCIAdditionalFlags.NUMBERED_DATA_PACKET if sequqence_number` in `__init__()`. 
In the other classes this could replace the `self.is_numbered` attribute. `self.additional_flags = APCIAdditionalFlags.NUMBERED_DATA_PACKET if is_numbered`